src/commands/synthetics/reporters/default.ts

```diff
@@ -139,9 +150,14 @@ const renderResultOutcome = (
   }
 
   if (test.type === 'browser') {
+    // We render the step only if the test hasn't passed to avoid cluttering the output.
     if (!result.passed && 'stepDetails' in result) {
-      // We render the step only if the test hasn't passed to avoid cluttering the output.
-      return result.stepDetails.map(renderStep).join('\n')
+      const criticalFailedStepIndex = result.stepDetails.findIndex((s) => s.error && !s.allowFailure) + 1
+
+      return [
+        ...result.stepDetails.slice(0, criticalFailedStepIndex).map(renderStep),
+        renderSkippedSteps(result.stepDetails.slice(criticalFailedStepIndex)),
```

Comment:
Nit? This will render a break line character even if there are no skipped steps to render:

```ts
console.log(['foo', undefined].join('\n'))
// "foo\n"
```

Reply:
Nice catch!